CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ pull request if there was one.
 Current
 -------
 ### Fixed:
+- [Fix: Add logic to rename aggregation to avoid name collision](https://github.com/yahoo/fili/issues/1095)
+    * Add renameIfConflicting logic for aggregations in BaseProtocolMetricMaker.
```

Comment:
This isn't a fix, this is a change.

Also, this is a breaking change, which we try to avoid, so put some bold text to tell people what they need to fix if this change breaks them (in this case, implementing the rename method)